stingray/fourier.py

```diff
@@ -1846,11 +1850,13 @@ def avg_pds_from_events(
     """
     if segment_size is None:
```

Comment:
If we have this line, shouldn't be better to set segment_size = None as the default value?

Reply:
Yeah, it would indeed make sense. However, this would be a breaking change and would require to make a number of modifications to the rest of the code as well.

Reply:
I understand, it makes sense